std_msgs/String req_sign
int16 req_num_1
int16 req_num_2
---
std_msgs/String res_result
